tests/flakyfunctional/cylc-take-checkpoints/00-basic/flow.cylc

```diff
@@ -0,0 +1,26 @@
+#!jinja2
```

Comment:
Methinks this is a rebase error.

Reply:
```suggestion
```

Reply:
I think the whole file is a rebase error?

Reply:
No.